shortcodes/project-stats.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * Creates a Line chart shortcode to display some Org project metrics such as number of opened issues
+ * THis shortcode relies on data recorded via wpcron gh_query_project_stats job.
+ * To make it work you want to:
+ *  1. Add a org/project slug into "Query Project Slug" in format: "$org_name#$project_name" (/wp-admin/edit-tags.php?taxonomy=ghactivity_query_project_slug&post_type=gh_query_record)
+ *  2. Wait or trigger gh_query_project_stats manually.
+ *
+ * Shortcode: ghactivity_project_stats
+ * Parameters:
+ *  string $org          Organisation name
+ *  string $project_name Project name
+ *  string $columns      Coma separated list of Project columns names needs to be displayed in graph.
+ *
+ * @package Ghactivity
+ */
+
+defined( 'ABSPATH' ) || die( 'No script kiddies please!' );
+
+add_shortcode( 'ghactivity_project_stats', 'output_project_stats' );
+
+/**
+ * Shortcode to display chart data of specific label
+ *
+ * @since 2.1.0
+ *
+ * @param array $atts Array of shortcode attributes.
+ */
+function output_project_stats( $atts ) {
+	$atts = shortcode_atts(
```

Comment:
It may be best to prefix this function to avoid potential conflicts with other plugins.